README.md

```diff
@@ -1,71 +1,70 @@
-## Preact bind-group
+## preact-bind-group
 
-#### What
+An event wrapper for preact (soon react) to centralize and simplify events management and state binding.
 
-`<BindGroup />` its a event wrapper for preact (soon react) to centralize and simplify events management and state binding.
+Check [the demo](https://codesandbox.io/s/wmpv1o4z8).
 
-#### Why
+### Why
 
 React/Preact forms are a bit cryptic because it leads developer to deal with too many language primitives to tie state with fields, care about events and duplicate this pattern for each field.
 
-#### How
+### How
 
-If you check the code u will see that is seeking into its children that contain `data-bind` attribute. Those children are intended to be "fields" with some change that you may track and some property that should be tied to the state value.
+preact-bind-group exposes a `<BindGroup>` component that looks for children that contain `data-bind` attribute which should be assigned to any element or component that emits an `onChange` event.
 
-BindGroup keeps an internal state to internally track these values.
 
-BindGroup supports any control field that uses `onChange` to deal with changes and it does by default, but additionally you can provide other event reporter with `data-event` as follows:
+### Get started
 
-```javascript
-<input data-bind="value0" data-event="onInput"/>
+Install it:
+
+```bash
+npm install --save-dev preact-bind-group
 ```
 
-#### Usage
+Import it it in your components and use it:
+
+```jsx
+import { render } from "preact";
+import { BindGroup } from "preact-bind-group";
+
+const App = () => (
+  <div>
+    <h3>BindGroup demo</h3>
+    <BindGroup watch={(...args) => console.log(args)}>
+      <div>
+        <input data-bind="value0" />
+      </div>
+      <div>
+        <input data-bind="value1" />
+      </div>
+    </BindGroup>
+  </div>
+);
+```
 
+### Watch callback
 
-```javascript
-<div>
-  <h3>BindGroup demo</h3>
-  <BindGroup watch={change => console.log(change)}>
-    <div>
-      <input data-bind="value0"/>
-    </div>
-    <div>
-      <input data-bind="value1"/>
-    </div>
-    <div>
-      <input data-bind="value2"/>
-    </div>
-    <div>
-      <input data-bind="value3"/>
-    </div>
-  </BindGroup>
-</div>
-```
-If you change any input value you should see this on the console:
+The `watch` callback receives an `{key: value}` object containing the changed property as its parameter.
```

Comment:
I would like cbk signature to be here  `({ [key: string]: any }, key: string) => void`.

Reply:
Ok